hw/acpi/aml-build.c

```diff
@@ -1673,12 +1675,13 @@ void acpi_get_pci_holes(Range *hole, Range *hole64)
 }
 
 
-bool acpi_get_mcfg(AcpiMcfgInfo *mcfg)
+bool acpi_get_mcfg(AcpiMcfgInfo *mcfg, AcpiPciBus *acpi_pci_bus)
```

Comment:
We need to do this differently. This should be sent in a list and we should build the table from the list.